### DISCUSSION TITLE
Alguém pode dar uma olhada no meu código?

### DISCUSSION THREAD

**Discussion by dlourennco** [frontendbr/forum#1622] [Mostre seu trabalho]
Olá! Estou estudando HTML, CSS e Javascript. Estava pesquisando e com a ajuda de alguns vídeos, fiz uma aplicação que mostra o tempo local e o background muda de acordo com o horário. Também acrescentei informações sobre o clima por meio de uma api gratuita do Darksky. Como melhorias estou pensando em fazer um banco de imagens para que ela seja alterada de tempos em tempos e também que a imagem do fundo esteja relacionada com o clima.

A aplicação aparentemente está funcionando com tudo certinho, mas acho que meu código está um pouco bagunçado, gostaria que alguém mais experiente pudesse dar uma olhada rápida no meu código e se possível me dar algumas direções de como reduzir o código e deixar ele com uma melhor leitura.

Segue o link do repositório: https://github.com/dlourennco/WeTime

Não sei se aqui seria o lugar correto para esse assunto, caso não, podem me direcionar?

Valeu pessoal.

**Comment by developerfred** [comment]
Em relação ao banco de imagem aconselho a você usar a api do unplash.

Review
- [ ] https://github.com/dlourennco/WeTime/issues/1 
- [ ] https://github.com/dlourennco/WeTime/issues/2
- [ ] https://github.com/dlourennco/WeTime/issues/3

um abraço.
